releases/v1.26.0

```diff
@@ -0,0 +1,76 @@
+# Highlights
+
+## Nexus (Pre-Release)
+
+Note: This feature requires a server version 1.25+ and must be enabled. For self hosted [see](https://github.com/temporalio/temporal/blob/main/docs/architecture/nexus.md#trying-nexus-out), for Temporal Cloud Nexus support is available as public preview.
+
+Temporal Nexus is a new feature of the Temporal platform designed to connect durable executions across team, namespace, region, and cloud boundaries. It promotes a more modular architecture for sharing a subset of your team’s capabilities via well-defined service API contracts for other teams to use, that abstract underlying Temporal primitives, likeWorkflows, or execute arbitrary code.
+
+Learn more at [temporal.io/nexus](https://temporal.io/nexus).
+
+More Java specific docs and samples coming soon.
+
+## Update-With-Start **(Private Preview)**
+
+Note: This feature requires a server version 1.24+ and must be enabled. For self hosted you can set `frontend.enableExecuteMultiOperation` for Temporal Cloud please reach out to your Temporal account team or [Temporal Support Team](https://docs.temporal.io/cloud/support#support-ticket) to enable update with start in your namespace.
+
+This release add support for Workflow Update-With-Start. Update-With-Start allows users to send a Workflow Update request along with a Start Workflow request in the same RPC. Users can think of Update-With-Start as analogous to Signal-With-Start except for Updates instead of Signals. To make a Update with start request users can call `WorkflowClient.updateWithStart`.
+
+## Workflow Init
+
+Added a new annotation `@WorkflowInit` users can add to the constructor of their workflow implementations. The constructor annotated with this annotation is called when a new workflow instance is created. It will be called before the the main workflow method and any signal or update handlers. The constructor must be public and take the same arguments as the workflow method. All normal workflow constraints also apply in the constructor.
+
+## Upsert Memo
+
+Added a new Workflow API `Workflow.upsertMemo` . This allows users to modify a workflows memo field inside the workflow. Normal determinism constraints apply to `upsertMemo`.
+
+## User Metadata (Public Preview)
+
+Note: This feature requires a server version 1.25+
+
+This release add support for user metadata inside Workflows. This lets users set custom metadata on Workflow executions and certain APIs inside workflow that will be visible on the UI.
```

Comment:
```suggestion
This release adds support for user metadata inside Workflows. This lets users set custom metadata on Workflow executions that will be visible in the UI.
```

How should we phrase the "and certain APIs inside workflow" bit? I wasn't sure exactly how to change it; it felt like it wasn't related to the UI.

Reply:
It is metadata on certain APIs that will be on the UI. I tried to use punctuation to make it clearer